great_expectations/cli/checkpoint_script_template.py

```diff
@@ -19,9 +19,9 @@
 import sys
 
 from great_expectations.checkpoint.types.checkpoint_result import CheckpointResult
-from great_expectations.data_context import DataContext
+from great_expectations.util import get_context
 
-data_context = DataContext(context_root_dir="{1:s}")
+data_context = get_context(context_root_dir="{1:s}")
```

Comment:
```suggestion
data_context: FileDataContext = get_context(context_root_dir="{1:s}")
```
Should we annotate this since we are annotating the `CheckpointResult`?

Reply:
sure! doing that now